src/lib/message/L2ToL1MessageNitro.ts

```diff
@@ -61,6 +61,8 @@ const ASSERTION_CREATED_PADDING = 50
 // expected number of L1 blocks that it takes for a validator to confirm an L1 block after the node deadline is passed
 const ASSERTION_CONFIRMED_PADDING = 20
 
+const l2BlockRangeCache: { [key in number]: number[] } = {}
```

Comment:
This needs to also be grouped by parent chain

Reply:
My bad, needs to be grouped by chain, cause one parent chain (e.g. Arb Goerli) can have many Orbit chains